360-Videos-Metadata/360VideosMetadata.py

```diff
@@ -828,8 +832,30 @@ def main():
                       action="store_true",
                       help="injects spherical metadata into a MP4/WebM file, "
                            "saving the result to a new file")
+    parser.add_option('-s', '--stereo',
+                      type='choice',
+                      action='store',
+                      dest='stereo',
+                      choices=['mono', 'over-under', 'left-right',],
+                      default='mono',
+                      help='Environment to run on',)
+
     (opts, args) = parser.parse_args()
 
+    # Configure inject xml.
+    additional_xml = ""
+    if opts.stereo == 'over-under':
+      additional_xml = spherical_xml_contents_over_under
+
+    if opts.stereo == 'left-right':
+      additional_xml = spherical_xml_contents_left_right
+
+    global spherical_xml
```

Comment:
Can we avoid the use of a global? It's tough to track how it gets used.

Instead we should only compute spherical_xml when injecting below, and then pass that to InjectMetadata(), which can then pass it to ProcessMKV() and ProcessMpeg4(), which in turn would call mpeg4_add_spherical()

You may have to refactor those few methods a bit, but I'd strongly prefer you do over using a global.